required/graphics/grfguide.tex

```diff
@@ -232,7 +232,8 @@ \subsubsection{Using predefined colours}
 
 This is a \emph{declaration}, like |\bfseries|. It changes the current
 colour to \m{name} until the end of the current group or
-environment.
+environment. It does not automatically start horizontal mode so typically at
+the begin of a paragraph it should be used in combination with a |\leavevmode|.
```

Comment:
```suggestion
the begin of a paragraph it should be used in combination with a |\leavevmode|.
```
```suggestion
the start of a paragraph it should be used in combination with a |\leavevmode|.
```

Reply:
Also, in the previous line, better with a comma before "or".